src/jit/lower.cpp

```diff
@@ -1423,13 +1512,15 @@ void Lowering::LowerCall(GenTree* node)
         BlockRange().InsertBefore(insertionPoint, std::move(resultRange));
 
         call->gtControlExpr = result;
+        ContainCheckRange(std::move(resultRange));
```

Comment:
This should be done before inserting the range, as it is logically invalidated by the earlier `std::move`.